synapse/federation/transport/server.py

```diff
@@ -542,6 +543,26 @@ async def on_PUT(self, origin, content, query, room_id, event_id):
         return 200, content
 
 
+class FederationMakeKnockServlet(BaseFederationServlet):
+    PATH = "/make_xyz.amorgan.knock/(?P<room_id>[^/]*)/(?P<user_id>[^/]*)"
+
+    PREFIX = FEDERATION_UNSTABLE_PREFIX
```

Comment:
We had a brief chat about this in #synapse-dev but I find it odd that this ends up as `/unstable/make_xyz.amorgan.knock/.../...`, I would have expected `/unstable/xyz.amorgan.knock/make_knock/.../...` which is a bit duplicative, but not sure what else to suggest.

(This also goes for `send_knock`.)

Reply:
Should be fixed by 338861f.